public/pages/workflow_detail/tools/tools.tsx

```diff
@@ -92,49 +88,37 @@ export function Tools(props: ToolsProps) {
       !isEmpty(searchPipelineErrors)
     ) {
       if (!isEmpty(opensearchError)) {
-        setCurErrorMessages([opensearchError]);
+        props.setCurErrorMessages([opensearchError]);
       } else if (!isEmpty(ingestPipelineErrors)) {
-        setCurErrorMessages([
+        props.setCurErrorMessages([
           'Data not ingested. Errors found with the following ingest processor(s):',
           ...Object.values(ingestPipelineErrors).map((ingestPipelineError) =>
             formatProcessorError(ingestPipelineError)
           ),
         ]);
       } else if (!isEmpty(searchPipelineErrors)) {
-        setCurErrorMessages([
+        props.setCurErrorMessages([
           'Errors found with the following search processor(s)',
           ...Object.values(searchPipelineErrors).map((searchPipelineError) =>
             formatProcessorError(searchPipelineError)
           ),
         ]);
       }
     } else {
-      setCurErrorMessages([]);
+      props.setCurErrorMessages([]);
```

Comment:
`curErrorMessages` was localized to `Tools` since that's where it was used. Since it is being moved to a higher level component, and errors are no longer being used here at all, let's move all of the error logic where `curErrorMessages` is defined & used.